microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Verify your Microsoft Edge Partner Center program information
+description: Learn how to verify your Microsoft Edge program information when your company enrolls in a new Partner Center program to publish extensions to the Microsoft Edge Add-ons store.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/31/2022
+---
+# Verify Microsoft Edge program information when you enrol in a new Partner Center program
```

Comment:
Coordinate the TOC title, title: in metadata, and h1 (#) title.  fix "enrol" typo in title.  Design rule: the TOC title is allowed to delete some words, but not allowed to use different words than top-of-page title.
rule: title in metadata at top must be same as h1 (#) title